src/odemis/model/_components.py

```diff
@@ -634,6 +634,16 @@ def __init__(self, name, role, **kwargs):
         # To provide some rough idea of the step size when changing focus
         self.depthOfField = _vattributes.FloatContinuous(1e-6, range=(0, 1e9),
                                                          unit="m", readonly=True)
+
+        # Microscope spot size. This is equivalent to the standard deviation of
+        # the Gaussian approximation of a fluorescence microscope point spread
+        # function, measured in number of pixels of the camera. This value can
+        # be used as a characteristic size parameter when filtering an image
+        # prior to spot detection. To convert this to the FWHM diameter of a
+        # spot multiply by `2 * sqrt(log(4)) ≈ 2.3548`
+        self.spotSize = _vattributes.FloatContinuous(1, range=(0, 1e9),
+                                                     unit="px", readonly=True)
```

Comment:
I'd suggest to use `PSFSize` as name then - I think this is more clear.

Similarly, I prefer to keep the unit in pixels. This way it does not matter whether someone wants to use the PSFSize in the 'object' or 'image' plane. If you provide it in meters, you always have to specify which observation plane you are referring to.

Thanks about the binning. It should change with binning. I suggest using `pxs = mdf[_metadata.MD_PIXEL_SIZE][0]` instead of `pxs = self.pixelSize.value[0]`. Note that this will then also change the behaviour of the `depthOfField` VA, but that should also be dependent on binning, correct?

Reply:
Pretty much any name which is not already used in Odemis is fine for me. That said, to be picky, I'd suggest not using an abreviation. Could we call it `pointSpreadFunctionSize`? Or `pointSpreadSize`? 

I'm not sure what you mean by object or image plane. Do you mean that if it's in meters, it would not be clear as to whether it's converted to pixels based on sensor pixel size or image pixel size? I think that's indeed a little unclear. One way would be to document it. The advantage is that it's binning independent. Otherwise, indeed, we'd need to update whenever the binning changes.

The idea of the depthOfField is to represent a minimum distance to move the *focus* so that the image is "sensibly different". Its main use is to define the smallest step size for the autofocus. I guess it should in theory take into account binning, as a large pixel size would mean you notice less the difference. I *think* that it's be fine to update its behaviour, but we might want to double check that the autofocusing of some system don't go too wrong (I don't expect it, because we don't usually running autofocus with a large binning!!)